feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewScreen.kt

```diff
@@ -89,7 +94,17 @@ fun PreviewScreen(
         Text(text = stringResource(R.string.camera_not_ready))
     } else if (previewUiState.cameraState == CameraState.READY) {
         Box(
-            modifier = Modifier.fillMaxSize()
+            modifier = Modifier.fillMaxSize().pointerInput(Unit) {
```

Comment:
You can use the rememberTransformableState for pinch zoom gestures.

```
val transformableState = rememberTransformableState(onTransformation = { zoomChange, _, _ ->
        viewModel.setZoomRatio(zoomChange)
    })
    
...
    
modifier = Modifier.fillMaxSize().transformable(state = transformableState),
```

https://developer.android.com/jetpack/compose/touch-input/pointer-input/multi-touch

Reply:
Done. Works like a charm